integration-tests/load/vrfv2plus/config.toml

```diff
@@ -1,14 +1,44 @@
 
 [Common]
-node_funds = 0.1
-is_native_payment = false
 minimum_confirmations = 3
-sub_funds_link = 10
-sub_funds_native = 1
+
+[NewEnvConfig]
```

Comment:
Maybe we can name it more reasonably, it's some common funding vars? Maybe we should move `number_of_sub_to_create` to the test groups or vary it in `Load` group?

Reply:
yeah, forgot to move `number_of_sub_to_create` to test groups, will do